### DISCUSSION TITLE
I stopped my subscription to Copilot, due to its unreliability

### DISCUSSION THREAD

**Discussion by lheinbokel** [community/community#40207] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

<img width="1937" alt="image" src="https://user-images.githubusercontent.com/46933003/204118444-fc5a7651-21c4-41d4-9de4-65d41c2a5810.png">

My Copilot stopped working again. That already happened yesterday, but the output in my VS Code was different. Yesterday it was due to some missing token and error code 403, today the error message looks like the one attached in the screenshot. 

I will consider to re-subsubscribe once these issues are fixed

**Comment by davecheney** [comment, marked as answer]
@lheinbokel I am sorry you've had such a poor experience recently. The difficulties over the weekend were our fault as we tried to roll out some new features. Please accept my apologies on behalf of the wider Copilot team.

The difficulty resolving `api.github.com` is unrelated; it is likely an error caused by your local network configuration.

Having said that, I completely understand if you wish to cancel your Copilot subscription, you can contact support https://support.github.com/contact who can assist you with that.
